bin/release-build.sh

```diff
@@ -11,10 +11,12 @@ if [[ $# != 1 ]]; then
   exit 1
 fi
 
+rpm_log_file="/build/logs/${1}_rpm.log"
 log_file="/build/logs/${1}.log"
-nohup time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir ${1%%-*} > $log_file 2>&1 &
-echo "${1} release build kicked off, see log @ $log_file ..."
-
 container_log_file="/build/logs/${1}_container.log"
-nohup time ${BUILD_DIR}/bin/container-build.sh ${1} > $container_log_file 2>&1 &
-echo "${1} release container build kicked off, see log @ $container_log_file ..."
+
+( nohup time ${BUILD_DIR}/bin/rpm-build.sh -t release -r ${1} > $rpm_log_file 2>&1;
```

Comment:
Why doesn't this one do && at the end like the nightly build?

Reply:
release build script has '-e' set, so it will not proceed if the previous command fails anyway.